ceph.spec.in

```diff
@@ -323,6 +324,8 @@ Requires:	python-flask
 %if 0%{?suse_version}
 # python-Flask for powerdns
 Requires:      python-Flask
+# libopenssl for civetweb https support
+Requires:      libopenssl1_0_0
```

Comment:
A `BuildRequires` installs a package and its dependencies in the build environment (i.e. on the build worker in OBS/IBS), while a `Requires` does the same in the runtime environment (e.g. customer machine where the product is in production). 

My knowledge of spec files is limited, too, I guess. How does the `BuildRequires: openssl-devel` cause `libopenssl1_0_0` to be added at runtime?